osu.Game/GameModes/Menu/ButtonSystem.cs

```diff
@@ -282,6 +282,9 @@ protected override void Update()
             //if (OsuGame.IdleTime > 6000 && State != MenuState.Exit)
             //    State = MenuState.Initial;
 
+            if (state == MenuState.Exit && osuLogo.Alpha <= 0)
+                Game.Exit();

```

Comment:
this shouldn't be required (exit happens in OsuGame.cs line 124)

![devenv_2016-10-12_11-34-33](https://cloud.githubusercontent.com/assets/191335/19295640/e38400e0-9070-11e6-894c-0882a0da33f8.png)